nixos/modules/programs/wayland/hyprland.nix

```
@@ -56,6 +56,7 @@ in
       services.displayManager.sessionPackages = [ cfg.package ];
 
       xdg.portal = {
+        enable = lib.mkDefault true;
```

```suggestion
        enable = true;
```
we probably want to confirm users when they are disabling that and that it isn't the recommended setup

---

I agree, and eval for some reason is failing?